lib/travis/api/v3/queries/user_builds.rb

```diff
@@ -0,0 +1,18 @@
+module Travis::API::V3
+  class Queries::UserBuilds < Query
```

Comment:
We generally have a correspondence of queries to resources to models, so I do think this should go into the the Builds query instead (ie `Queries::Builds#for_user` instead of `Queries::UserBuilds#find`).